carbon-design-system/carbon-addons-iot-react
# fix(datetimepicker): needs to use the correct locale in the card

Closes #

**Summary**

- fixes a bug where the cards were not passing locale down to the CardToolbar to render the DateTimePicker so the calendar was in the wrong format

**Change List (commits, features, bugs, etc)**

- fix(Card): pass through the locale to the card
- fix(CardToolbar): localize the dates shown in the human readable date picker

**Acceptance Test (how to verify the PR)**

- Verify if you look at the card range picker story and switch the locale, that the human readable date and the Calendar show correct locales in the Absolute Range Picker

**Regression Test (how to make sure this PR doesn't break old functionality)**

- Verify in english



**Things to look for during review**
